userspace/libsinsp/threadinfo.h

```diff
@@ -169,7 +169,15 @@ class SINSP_PUBLIC sinsp_threadinfo
 
 		if(fdt)
 		{
-			return fdt->find(fd);
```

Comment:
You can remove those changes now and update `m_oldname` when you set `fdinfo_name_changed` in the parser process event